src/Encryption/AesEncryption.php

```diff
@@ -50,10 +47,10 @@ public function __construct($aesKey)
      * @param string $input
      * @return string
      */
-    public function encrypt($input)
+    public function encrypt(string $input): string
     {
         return base64_encode(
-            mcrypt_encrypt(MCRYPT_RIJNDAEL_256, $this->secureKey, $input, MCRYPT_MODE_ECB, $this->iv)
+            openssl_encrypt ($input, self::CHIPPER, $this->secureKey, 0, self::IV)
```

Comment:
remove the space before after method name `openssl_encrypt (....)` change to `openssl_encrypt(....)`

Reply:
I found out spaces issue :) My IDE is adding variable hint therefor it makes it hard to count/see spaces.